bot.test.js

```diff
@@ -24,7 +26,7 @@ describe('bot.getReply', () => {
       [['x\ny'], 'x\ny'], // line splitting happens a layer below, we gucci
     ];
 
-    for (let i = 0; i < tests.length; i++) {
+    for (let i = 0; i < tests.length; i += 1) {
```

Comment:
What

Reply:
eslint

Reply:
Specifically https://eslint.org/docs/rules/no-plusplus if you're curious their justification. (automatic semicolon insertion ftw)